TLM/TLM/Resources/Translations/Menu.csv

```diff
@@ -11,11 +11,11 @@
-"Label:Incoming demand","Eingehende Nachfrage","Incoming demand","Demanda entrante","Demande entrante","Bejövő kereslet","Label:Incoming demand","入っていく需要","들어오는 수요","Inkomende vraag","Zapotrzebowanie - przychodzące","Demanda de entrada","Входящий спрос","入行需求","进货需求","Incoming demand","Geliş talebi"
+"Label:Outgoing demand","Ausgehende Nachfrage","Outgoing demand","Demanda saliente","Demande sortante","Kimenő kereslet","","出ていく需要","나가는 수요","Uitgaande vraag","Zapotrzebowanie - wychodzące","Demanda de saída","Исходящий спрос","出行需求","出货需求","Outgoing demand","Çıkış talebi"
+"Label:Incoming demand","Eingehende Nachfrage","Incoming demand","Demanda entrante","Demande entrante","Bejövő kereslet","","入っていく需要","들어오는 수요","Inkomende vraag","Zapotrzebowanie - przychodzące","Demanda de entrada","Входящий спрос","入行需求","进货需求","Incoming demand","Geliş talebi"
 "Tooltip:Lane connector","Fahrspurverbinder","Lane connector","Conector de carriles","Connecteur de voies","Sávcsatlakozó","Connettore corsia","車線を接続","차선 연결","Rijstrookverbinder","Połącz pasy ruchu","Conector de faixas","Соединения полос движения","車道連接工具","车道连接","Lane connector","Şerit birleştiricisi"
 "Tooltip:Junction restrictions","Kreuzungsbeschränkungen","Junction restrictions","Restricciones del cruce","Restrictions des intersections","Csomópont korlátozások","Restrizioni incrocio","交差点の制限","교차로 규칙","Splitsingsbeperkingen","Ograniczenia na skrzyżowaniach","Restrições de junção","Ограничения на перекрёстках","交岔路口動線連接制定","路口管制","Junction restrictions","Kavşak kısıtlamaları"
-"Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Ctrl+Click: Separate turning lanes for entire junction \nAlt+Click: Separate turning lanes for segment","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Tooltip.Keybinds:Change lane arrows","Ctrl+Klik: Oddziel pasy skrętu we wszystkich segmentach skrzyżowania \nAlt+Klik: Oddziel pasy skrętu wybranego segmentu","Tooltip.Keybinds:Change lane arrows","Ctrl+Клик: Отдельные полосы для поворотов на всём перекрёстке \nAlt+Клик: Отдельные полосы для поворотов на выбранном участке дороги","Tooltip.Keybinds:Change lane arrows","Ctrl+点击: 整个路口的车道更改为单独行车导向箭头\nAlt+点击: 一段路口的车道更改为单独行车导向箭头","Ctrl+Click: Separate turning lanes for entire junction \nAlt+Click: Separate turning lanes for segment","Şerit oklarını değiştir"
-"Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Ctrl+Click: Quick setup","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Ctrl + Klik: Szybka instalacja","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Tooltip.Keybinds:Auto TL","Ctrl+Click: Quick setup","Tooltip.Keybinds:Auto TL"
```

Comment:
I'm pretty sure you've copy & pasted this. If there is no translation on Crowdin, there should be empty string(or I'm missing something), but not translation key :wink: